apps/wallet/src/components/external-canisters/CanisterMonitorDialog.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <VDialog
```

Comment:
recently i tried introducing a composable that can be used to make Form handling more uniform in the codebase, could you take a look [here](https://github.com/dfinity/orbit/blob/main/apps/wallet/src/components/external-canisters/CanisterCallDialog.vue#L19) and try and align this dialog and form implementation with it?

Reply:
done